### DISCUSSION TITLE
Unable to upgrade Netbox 3.3.5 -> 3.4.8

### DISCUSSION THREAD

**Discussion by aakozlov85** [netbox-community/netbox#12332] [General]
Hi!
I am trying to upgrade my Netbox 3.3.5 to the latest stable release.
I am using upgrade.sh script for that, following official documentation.
At migrations step i am getting error:
django.db.utils.IntegrityError: could not create unique index "dcim_device_unique_name_site_tenant"
DETAIL:  Key (lower(name::text), site_id, tenant_id)=(n55-m16up, 1, 1) is duplicated

Please advise for my next steps.

**Comment by candlerb** [comment]
Thie is described in the [release notes](https://docs.netbox.dev/en/stable/release-notes/version-3.4/#breaking-changes) for v3.4 under "Breaking Changes":

> Device and virtual machine names are no longer case-sensitive. Attempting to create e.g. "device1" and "DEVICE1" within the same site will raise a validation error.

In your case it looks like you have two devices in the same site with names which conflict when downcased; for example they might be

* n55-m16up
* N55-m16up

Your solution is either to delete one of these devices, or to rename one of them (e.g. to "N55-m16up-1")

**Reply by aakozlov85** [reply]
yes, you are right.
we have 2 devices with the same name
n55-m16up and N55-M16UP

thank you!
